docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -114,33 +133,32 @@ Play is being rebuilt and progressively rolled out to more users. This section e
 
 :::note
-[Inbound Connectors](/components/connectors/connector-types.md#inbound-connectors) and [Connectors in hybrid mode](/guides/use-connectors-in-hybrid-mode.md) do not connect to external systems and must be completed manually.
 [Start events with forms](/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md#embed-form-in-start-event) will be completed without the form being shown.
 [Decision table rule](/components/modeler/dmn/decision-table-rule.md) evaluations are not viewable. However, they can be inferred from the output variable.
 :::
 
-There are some bugs related to Play’s architecture. These will be resolved when Play is integrated with Camunda development clusters.
-
-- **Timer events** sometimes fail silently. Try refreshing the page.
-- Deployment sometimes fails with a network error in the **Output** panel of the **Implement** mode. If these errors repeat, let your Play cluster expire and try again.
-
-## Alpha feature
-
-Play is an alpha feature for a few reasons:
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
 
-- Play runs on community-built projects, as described in the [Zeebe-Play repository](https://github.com/camunda-community-hub/zeebe-play).
+## Configuration for Self-Managed
 
-- Play is run on completely isolated Camunda-hosted infrastructure from the core SaaS or Self-Managed Camunda deployment. It can only receive information from Web Modeler and can only communicate externally using Connectors and the user-defined secrets in the secret store or BPMN diagram.
+When opening Play in Self-Managed, you will be asked to fill in the details of your cluster.
 
-  :::note
-  Play is not authenticated, so anyone with the URL can access it. You should not submit personal or confidential information to Play.
-  :::
+Find information below, as well as example values for the simple docker setup.
 
-- To mitigate this risk, a 34-character randomly generated UUID is in the URL. Each session lasts approximately 20 minutes, and the Play environment and its data are automatically deleted at the end of each session. You can reset the session timer by re-opening Play.
+![play cluster config](img/play-cluster-configuration.png)
 
-For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+| Name              | Description                                  | Example value                                                                     |
+| ----------------- | -------------------------------------------- | --------------------------------------------------------------------------------- |
+| Cluster endpoint  | Address where your cluster can be reached.   | `http://zeebe:26500`                                                              |
```

Comment:
I think these Names are inconsistent with what we have and also not as easy as they could be (obviously, we'd also need to change that in the deployment modal). 
1. Cluster Endpoint should be Zeebe Gateway Address or simply Zebee Address as this is the endpoint for Zeebe and not the entire cluster.
2. OAuth audience, I think should be Zeebe token audience? This makes it less confusing for which service this is
<img width="562" alt="image" src="https://github.com/user-attachments/assets/4242005b-ddab-4d12-8f39-ce293b48c942">
3. Besides I think this should be grouped accordingly (both here and in the modal. All Zeebe related things should come first, then the other stuff. 
4. There also seems to be some issues with capitalization. Eg., Zeebe REST URL rather than  Zeebe rest url ?
WDYT?

Reply:
though this might not be a quick update, as some of these fields are shared. I think we would have to update deployment modal as well 😅 and a lot of tests will fail....

Reply:
@huygur let's discuss in the standup. I don't think changing Play's modal would constitute a new feature. So I would argue it's ok to "fix" this between alpha and final release. But, yes, let's discuss this in the team

Reply:
Just as a note also here;

we decided not to squeeze in a rushed change in the Modal, therefore docs will also stay like this. 
it will be improved some time between 8.6-alpha5 - 8.6 release.